drivers/tty/serial/8250/8250_port.c

```diff
@@ -3224,6 +3224,7 @@ static void serial8250_config_port(struct uart_port *port, int flags)
 
 	register_dev_spec_attr_grp(up);
 	up->fcr = uart_config[up->port.type].fcr;
+	serial8250_enable_ms(port); // test-only
```

Comment:
This is not just test only - without this, serial will not work (but I do not know anymore why exactly).

```suggestion
	serial8250_enable_ms(port);
```

(AFAIR: This was about RTS/CTS not working)